lib/CLI.js

```diff
@@ -13,8 +13,10 @@ const logger = require('./logger');
 const { applyLoggingOptions } = require('./configuration');
 const { spawn } = require('./childProcess');
 
+const dreddOptions = require('./options.json');
```

Comment:
I don't think the extension here is nessecery, in same way how you don't specify `.js` on the other sources. We can omit for consistency.

```suggestion
const dreddOptions = require('./options');
```

Reply:
Oh, I thought it is necessary for `.json`. I don't know why.